src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -36,7 +36,13 @@
      * exit value when an unrecoverable {@link UserException} occurs
      */
     private static final int USER_EXCEPTION_EXIT_VALUE = 2;
-    
+
+    /**
+     * exit value when an issue with the commandline is detected, ie {@link UserException.CommandLineException}.
+     * This is the same value as picard uses.
+     */
+    private static final int COMMANDLINE_EXCEPTION_EXIT_VALUE = 1;
```

Comment:
Do we want this to be the same as for all other exceptions?  I would bump the `ANY_OTHER_EXCEPTION_VALUE` to be 3 if this going to claim 1.